brooklyn-docs/guide/start/blueprints.md

```diff
@@ -6,44 +6,83 @@ children:
 - { section: Launching from a Blueprint, title: Blueprint } 
 ---
 
-<div style="width: 100%; display: block; background-color: #CC9966; margin-bottom: 2px;  padding: 50px 30px 50px 80px;" >
-  <h3>NOTE</h3>
-  <div>
-  The structure of Brooklyn's repositories is changing at present (Jan 2016). Until this is complete 
-  please obtain the "br" command line tool from <a href="https://github.com/brooklyncentral/brooklyn-cli">Brooklyn Central</a>
-  </div>
-</div>
+Blueprints are descriptors or patterns which describe how Apache Brooklyn should deploy applications. Blueprints are written in [YAML](https://en.wikipedia.org/wiki/YAML){:target="_blank"} and all of the entities available are defined in the __[Brooklyn Catalog](../../learnmore/catalog/)__.
 
 ## Launching from a Blueprint
 
-We'll start by deploying an application with a simple YAML blueprint containing a Tomcat server.
+We'll start by deploying an application with a simple YAML blueprint containing an [Apache Tomcat](https://tomcat.apache.org/){:target="_blank"} server.
 
-Copy the blueprint below into a text file, "myapp.yaml", in your workspace, but *before* you create an application with 
-it, modify the YAML to specify the location where the application will be deployed.  (Note, to copy the file you can
-hover your mouse over the right side of the text box below to get a Javascript "copy" button.)
+Copy the blueprint below into a text file, "myapp.yaml", in your workspace (Note, to copy the file you can
+hover your mouse over the right side of the text box below to get a Javascript "copy" button).
 
 {% highlight yaml %}
 name: Tomcat
 location:
```

Comment:
Aled pointed out that if there was something invalid in the location such as a <placeholder> then the error message from Brooklyn would be more invalid location rather than invalid YAML which is more sensible